static/app/utils/discover/eventView.tsx

```diff
@@ -38,12 +38,12 @@ import {
   DISPLAY_MODE_FALLBACK_OPTIONS,
```

Comment:
The changes in this file is almost entirely removing the `queryDataset` param from EventView. the discover component instead handles figuring which `dataset` value to pass to Eventview.